packages/validator/tests/oas.spec.ts

```diff
@@ -41,15 +41,15 @@ describe('OASValidator', () => {
 
       const result = await validator.verify(input);
 
-      expect(Object.keys(result)).toHaveLength(0);
+      expect(result).toMatchObject([]);
     });
 
     it('should successfully validate valid oas v3 document (spoonacular, json)', async () => {
       const input: OpenAPIV3.Document = spoonacularOas as OpenAPIV3.Document;
 
       const result = await validator.verify(input);
 
-      expect(Object.keys(result)).toHaveLength(0);
+      expect(result).toMatchObject([]);
```

Comment:
Please fix the same issue in other places as well

```suggestion
      expect(result).toEqual([]);
```